File: sfhwebservice/pojos/piezadental.php
<?php
class PiezaDental
{
	public $idPieza;
	public $idTratamientoDental;
	public $idOrdenLaboratorio;
	public $idTipoPieza;
	public $nombrePieza;
	public $descripcionPieza;
	public $periodoPieza;

	function initClass($idPieza, $idTratamientoDental, $idOrdenLaboratorio, $idTipoPieza, $nombrePieza
					 $descripcionPieza, $periodoPieza)
	{
		$this->idPieza = $idPieza;
		$this->idTratamientoDental = $idTratamientoDental;
		$this->idOrdenLaboratorio = $idOrdenLaboratorio;
		$this->idTipoPieza = $idTipoPieza;
		$this->nombrePieza = $nombrePieza;
		$this->descripcionPieza = $descripcionPieza;
		$this->periodoPieza = $periodoPieza;
	}
}

?>

File: SFH-Modulo-Escritorio Ver Anteriores/SFH-Modulo-Escritorio v 1.3/ObjectsBeans/Piezadental.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ObjectsBeans
{
   public class Piezadental
    {
        #region Campos
        private int idPieza;
        private int idTratamientoDental;
        private int idOrdenLaboratorio;
        private int idTipoPieza;
        private string nombrePieza;
        private string descripcionPieza;
        private int periodoPieza;
        #endregion

        #region Constructor
        public Piezadental() { }
        #endregion

        #region Propiedad

        public int IdPieza
        {
            get { return idPieza; }
            set { idPieza = value; }
        }

        public int IdTratamientoDental
        {
            get { return idTratamientoDental; }
            set { idTratamientoDental = value; }
        }

        public int IdOrdenLaboratorio
        {
            get { return idOrdenLaboratorio; }
            set { idOrdenLaboratorio = value; }
        }

        public int IdTipoPieza
        {
            get { return idTipoPieza; }
            set { idTipoPieza = value; }
        }

        public string NombrePieza
        {
            get { return nombrePieza; }
            set { nombrePieza = value; }
        }

        public string DescripcionPieza
        {
            get { return descripcionPieza; }
            set { descripcionPieza = value; }
        }

        public int PeriodoPieza
        {
            get { return periodoPieza; }
            set { periodoPieza = value; }
        }
        #endregion
    }
}
